pkg/kubelet/dockershim/remote/docker_service.go

```diff
@@ -138,19 +139,32 @@ func (d *dockerService) ContainerStatus(ctx context.Context, r *runtimeApi.Conta
 }
 
 func (d *dockerService) ExecSync(ctx context.Context, r *runtimeApi.ExecSyncRequest) (*runtimeApi.ExecSyncResponse, error) {
-	return nil, fmt.Errorf("not implemented")
+	stdout, stderr, err := d.runtimeService.ExecSync(r.GetContainerId(), r.GetCmd(), time.Duration(r.GetTimeout())*time.Second)
```

Comment:
Right, it seems to be missing from the signature and request struct… I'm wondering how the execed process gets it